src/Core/Managed/Shared/TelemetryClient.cs

```diff
@@ -293,11 +293,10 @@ public void Track(ITelemetry telemetry)
             if (this.IsEnabled())
             {
                 this.Initialize(telemetry);
-                
-                this.WriteTelemetryToDebugOutput(telemetry);
```

Comment:
@dimazaid - you can write telemetry processor to filter out data. This PR introduces the new prefix in output window that Diagnostics Hub should ideally pick up and visualize somehow